wallet-new/server/Main.hs

```diff
@@ -11,14 +12,12 @@ import qualified Cardano.Wallet.LegacyAction as Legacy
 import           Cardano.Wallet.Server.CLI (ChooseWalletBackend (..),
                      WalletStartupOptions (..), getWalletNodeOptions)
 
-
 -- | The main entrypoint for the Wallet.
 main :: IO ()
 main = withCompileInfo $ do
     WalletStartupOptions cArgs wArgs <- getWalletNodeOptions
     let lArgs = loggingParams "node" cArgs
     let nArgs = NodeArgs { behaviorConfigPath = Nothing }
-
     putText "Wallet is starting..."
```

Comment:
No big deal, but let's leave it untouch. I can already foresee the very confusing git merge conflicts incoming here :upside_down_face:

Reply:
Oh, right, sorry that slipped through!